play-services-fido/core/src/main/kotlin/org/microg/gms/fido/core/RequestHandling.kt

```diff
@@ -213,21 +217,23 @@ fun getApplicationName(context: Context, options: RequestOptions, callingPackage
 }
 
 fun getApkKeyHashFacetId(context: Context, packageName: String): String {
-    val digest = context.packageManager.getFirstSignatureDigest(packageName, "SHA1")
+    // Default: SHA-256
+    val digest = context.packageManager.getFirstSignatureDigest(packageName, "SHA-256")
         ?: throw RequestHandlingException(NOT_ALLOWED_ERR, "Unknown package $packageName")
     return "android:apk-key-hash:${digest.toBase64(HASH_BASE64_FLAGS)}"
```

Comment:
This seems wrong. It should be either `android:apk-key-hash:<sha-1>` or `android:apk-key-hash-sha256:<sha-256>`, but not a mix.